### ISSUE TITLE
Editor auto-scrolls when typing 

### ISSUE THREAD

**Issue by TahaTesser** [invertase/zapp.run#145]
Operating System & Version: `macOS Ventura 13.4`
Browser & Version: `Version 114.0.5735.90 (Official Build) (arm64)`

### Description

When typing, the editor auto-scrolls. It makes it difficult to keep the code in the visible area when typing a few lines of code.

https://github.com/invertase/zapp.run/assets/48603081/3c330a93-fdd8-4708-a071-1def4bc274db

**Comment by Ehesp** [comment]
Erm ok I don't see this! Is there anything in console logs spitting errors out? What keybindings do you have enabled?

**Comment by TahaTesser** [comment]
@Ehesp 

> Is there anything in console logs spitting errors out?

I see this in the console


https://github.com/invertase/zapp.run/assets/48603081/1837dcc4-7ee2-4bb2-b341-edb57256a9ba


> What keybindings do you have enabled?


I've VS Code keybindings. Also tried switching to IntelliJ, same results. 
![Screenshot 2023-06-05 at 13 56 10](https://github.com/invertase/zapp.run/assets/48603081/e6b61036-8fbd-4370-9b2e-b1f1040ed7cd)

**Comment by YohanWadia** [comment]
Yeah exactly the same problem I am experiencing ... and I can vouch for all my beginner students are experiencing the same
